### PR TITLE
[WIP] Remove Schema and Doc generation from Jenkinsfile

### PR DESCRIPTION
**Pull request by robpal94**
## What did you change?
This PR removes the schema and doc generation, which are not used anymore and leads to build errors. 

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -48,8 +48,8 @@ pipeline {
       steps {
         sh('node --version')
         sh('npm run build')
-        sh('npm run build-schemas')
-        sh('npm run build-doc')
+        //sh('npm run build-schemas')
+        //sh('npm run build-doc')
```
Comment (Paulomart) [inline Jenkinsfile]: Please add spacing after //

**Comment by Paulomart** [comment]
I think we agreed on not removing the v7 Jenkinsfile. The job is disabled in Jenkins anyway.

**Comment by robpal94** [comment]
> I think we agreed on not removing the v7 Jenkinsfile. The job is disabled in Jenkins anyway.

@ElRaptorus

**Comment by ElRaptorus** [comment]
@Paulomart 
@5ebastianMeier and I discussed this and we would like to see those files gone, *because* they serve no purpose any longer.

**Pull request closed without merging**